internal/events/get.go

```diff
@@ -26,18 +26,29 @@ import (
 	"github.com/onflow/flow-cli/pkg/flowkit/services"
 )
 
-type flagsGenerate struct{}
+type flagsEvents struct {
+	Start   uint64 `flag:"start" info:"Block height start"`
+	End     uint64 `flag:"end" info:"Block height end"`
+	Last    uint64 `default:"1" flag:"last" info:"Fetch last number of block relative to latestBlocks. Will be ignored if --start set"`
+	Workers int    `default:"10" flag:"workers" info:"Number of workers to use when fetching events in parallel"`
+	Batch   uint64 `default:"250" flag:"batch" info:"Number of blocks to batch together when fetching events in parallel"`
+}
 
-var generateFlag = flagsGenerate{}
+var eventsFlags = flagsEvents{}
 
 var GetCommand = &command.Command{
 	Cmd: &cobra.Command{
-		Use:     "get <event_name> <block_height_range_start> <optional:block_height_range_end|latest>",
-		Short:   "Get events in a block range",
-		Args:    cobra.RangeArgs(2, 3),
-		Example: "flow events get A.1654653399040a61.FlowToken.TokensDeposited 11559500 11559600",
+		Use:   "get <event_name> (<event_name> <event_name>)",
```

Comment:
This is somewhat confusing, I actually from reading the help can't figure out what optionally second or third <event_name> means. I would strip that from the usage as it is an advance use case and document this in the docs more.